pytket/docs/circuit.rst

```diff
@@ -14,104 +14,108 @@ pytket.circuit
 
 .. automodule:: pytket._tket.circuit
     :members: fresh_symbol
-.. autoclass:: pytket._tket.circuit.Op
+.. autoclass:: pytket.circuit.Op
     :members:
-.. autoclass:: pytket._tket.circuit.Command
+.. autoclass:: pytket.circuit.Command
     :members:
-.. autoclass:: pytket._tket.circuit.BasisOrder
+.. autoclass:: pytket.circuit.BasisOrder
     :members:
-.. autoclass:: pytket._tket.circuit.CircBox
+.. autoclass:: pytket.circuit.CircBox
```

Comment:
This looks like a merge error; you have `CircBox` twice.

Reply:
Thanks for spotting, fixed now.